P20-P25.s

```diff
@@ -724,7 +724,7 @@ DECRM61		TS	R61CNTR
 #
 #	USTAR, MODIFIED FICTITIOUS STAR DIRECTION (1/2 UNIT VECTOR)
```

Comment:
Non-spotted typo.

There should be no space between `USTAR, MODIFIED` - i.e `USTAR,MODIFIED`. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/0576.jpg)